src/pages/ReimbursementAccount/RequestorStep.js

```diff
@@ -1,5 +1,103 @@
 import React from 'react';
-import {View} from 'react-native';
+import {View, ScrollView} from 'react-native';
+import styles from '../../styles/styles';
+import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
+import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
+import CONST from '../../CONST';
+import TextLink from '../../components/TextLink';
+import Navigation from '../../libs/Navigation/Navigation';
+import CheckboxWithLabel from '../../components/CheckboxWithLabel';
+import Text from '../../components/Text';
+import {goToWithdrawalAccountSetupStep} from '../../libs/actions/BankAccounts';
+import Button from '../../components/Button';
+import StatePicker from '../../components/StatePicker';
+import TextInputWithLabel from '../../components/TextInputWithLabel';
+import FixedFooter from '../../components/FixedFooter';
 
-const RequestorStep = () => <View />;
-export default RequestorStep;
+const RequestorStep = ({translate}) => (
```

Comment:
NAB, this is going to require `state` to hold all the form values so we should make it a class component to start + hook that stuff up, but not submit the form yet (or at least just make it a class component since it will have state). Could also wait to do this.

Reply:
Hah, yeah if you look at prior commits it was not a stateless component in earlier versions - I updated it to match Rory's PR since that how we merged it. 

I'm happy to update it in a future PR when we hook this up to the API

Reply:
Ouchies 🙃